docs/app.tsx

```diff
@@ -142,31 +170,62 @@ async function graph({
       // benchmark and node-version
       benchmark: [
         BENCHMARKS_ORDER[b.benchmark],
-        NODE_VERSIONS.indexOf(getNodeMajorVersionNumber(b.nodeVersion)),
-        b.nodeVersion,
+        NODE_VERSIONS.indexOf(getNodeMajorVersionNumber(b.runtimeVersion)),
+        b.runtimeVersion,
         b.benchmark,
       ].join('-'),
     }));
 
-  const nodeJsVersionCount = new Set(values.map(v => v.nodeVersion)).size;
+  const valuesBun = benchmarkResultsBun
+    .filter(
+      b =>
+        selectedBenchmarkSet.has(b.benchmark) &&
+        selectedBunVersionsSet.has(b.runtimeVersion)
+    )
+    .map(b => ({
+      ...b,
+      opsLabel: b.ops.toLocaleString('en-US'),
+      // artificical benchmark name to make sure its always sorted by
+      // benchmark and node-version
```

Comment:
node-version -> bun-version

These kinds of little mistakes happen to me too when copying code.